$NetBSD: patch-Makefile,v 1.3 2013/08/25 14:29:52 richard Exp $

* Fix build on systems lacking strverscmp.
* Remove CFLAGS/LDFLAGS passed in as MAKE_FLAGS
* Fix DESTDIR.

--- Makefile.orig	2011-06-24 14:25:27.000000000 +0000
+++ Makefile
@@ -31,13 +31,13 @@ OBJS=tree.o unix.o html.o xml.o hash.o c
 
 # Linux defaults:
 #CFLAGS=-ggdb -Wall -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
-CFLAGS=-O4 -Wall  -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
-LDFLAGS=-s
+#CFLAGS=-DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+#LDFLAGS=-s
 
 # Uncomment for FreeBSD:
 #CFLAGS=-O2 -Wall -fomit-frame-pointer
 #LDFLAGS=-s
-#OBJS+=strverscmp.o
+OBJS+=strverscmp.o
 
 # Uncomment for Solaris:
 #CC=cc
@@ -94,12 +94,12 @@ clean:
 	rm -f *~
 
 install: tree
-	install -d $(BINDIR)
-	install -d $(MANDIR)
+	install -d $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(MANDIR)
 	if [ -e $(TREE_DEST) ]; then \
-		install -s $(TREE_DEST) $(BINDIR)/$(TREE_DEST); \
+		install -s $(TREE_DEST) $(DESTDIR)$(BINDIR)/$(TREE_DEST); \
 	fi
-	install doc/$(MAN) $(MANDIR)/$(MAN)
+	install doc/$(MAN) $(DESTDIR)$(MANDIR)/$(MAN)
 
 distclean:
 	if [ -f tree.o ]; then rm *.o; fi
